### PR TITLE
Update test utils dependencies

### PR DESCRIPTION
**Pull request by dbacarel**
Signed-off-by: Daniele Bacarella <daniele.bacarella@here.com>

Thank you for contributing to harp.gl!

Before requesting a pull request, please remember to check the following documents:
* [contribution guidelines](https://github.com/heremaps/harp.gl/blob/master/CONTRIBUTING.md)
* [coding style](https://github.com/heremaps/harp.gl/blob/master/CODINGSTYLE.md)

If you are adding new functionality we would highly appreciate if you can describe what is the capability you are adding and even better if you can add some examples. Please also remember to add tests for it.

# CI Check

Our bots will check whether your PR can be directly integrated into the mainline. We have some internal integration tests running on the background, our bots will inform you of the next steps and someone from our team will take a look and help if needed!

And please do not forget to sign-off your commit! You can read more about DCO [here](https://julien.ponge.org/blog/developer-certificate-of-origin-versus-contributor-license-agreements/). But, in short, you just need to use `git commit -s` or append `--signoff` when you are committing to the repo.

Happy contributing!

### CODE REVIEW

**Comment by dbacarel** [comment]
`sinon`: ` 9.0.3` -> `9.2.2`
`@types/sinon`: `9.0.9` -> `9.0.10`
`mocha`: ` 7.2.0`  -> `8.2.1`
`ts-mocha`: ` 7.0.0` -> ` 8.0.0`
`@types/mocha`: ` 7.0.2` -> `8.2.0`
`@types/chai` : ` 4.2.12` -> `4.2.14`
`nightwatch`: ` 0.9.19 ` -> ` 1.5.1`
`@types/nightwatch`: ` 1.3.1` -> `1.3.2`
`geckodriver`: ` 1.20.0` -> `1.21.1`

File: yarn.lock [hunk 1]
```diff
@@ -8804,7 +8775,7 @@ simple-swizzle@^0.2.2:
   dependencies:
     is-arrayish "^0.3.1"
 
-sinon@^9.0.1, sinon@^9.0.2:
+sinon@^9.0.1:
```
Comment (FraukeF) [inline yarn.lock:8778]: just a minor: the two sinon versions could both be covered with 9.2.3, you could run yarn-deduplicate over the file to remove the obsolete

File: yarn.lock [hunk 2]
```diff
@@ -1635,11 +1644,16 @@
   dependencies:
     "@types/chai" "*"
 
-"@types/chai@*", "@types/chai@^4.2.11":
+"@types/chai@*":
```
Comment (FraukeF) [inline yarn.lock:1647]: chai 4.2.14 would cover both cases

File: yarn.lock (hunk 1 above)
Comment (dbacarel) [reply yarn.lock]: done

File: yarn.lock (hunk 2 above)
Comment (dbacarel) [reply yarn.lock]: done

**Review by FraukeF — DISMISSED** [review]

**Comment by dbacarel** [comment]
retest this please

**Review by FraukeF — APPROVED** [review]

**Pull request merged**